plugin.video.srmediathek/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.srmediathek" name="SR Mediathek" version="1.0.0" provider-name="membrane">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+		<import addon="script.module.libsr" version="1.0.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+		<language>de</language>
+		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+		<source>https://github.com/prof-membrane/plugin.video.srmediathek</source>
+        <summary lang="en">This is an add-on for http://sr-mediathek.de/.</summary>
+        <description lang="en">This add-on provides access to the German VOD platform http://sr-mediathek.de/.</description>
```

Comment:
License and source are already there. 

Could you let the missing forum thread/email slide for now? I want to publish the add-ons before the release of Krypton. I will add the missing fields in an update.